runtime/onert/core/include/compiler/train/TrainingInfo.h

```diff
@@ -42,6 +42,8 @@ class TrainingInfo
   ~TrainingInfo() = default;
 
   bool shouldTrain() const { return _has_info; }
+  const uint32_t batchSize() const { return _batch_size; }
```

Comment:
```suggestion
  uint32_t batchSize() const { return _batch_size; }
```
"const" is redundant when returning by value because it will be copied.